images/manageiq-base/container-assets/create_local_yum_repo.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+yum -y install createrepo_c
+createrepo /tmp/rpms
+yum -y remove createrepo_c
+
+cat > /etc/yum.repos.d/local_rpm.repo << EOF
+[local-rpm]
+baseurl=file:///tmp/rpms/$basearch
+name=Local yum repo
+enabled=1
+gpgcheck=0
+EOF
+
+dnf config-manager --setopt=manageiq-11-kasparov*.exclude=manageiq-* --save
```

Comment:
Should manageiq be hardcoded here?

Reply:
Right now, everything is hard-coded to use manageiq prefix. I'll open an issue to make it customizable.

Reply:
:confused: I mean ... it's in all the rpm names. If it needs to be configurable then we're going to need to change a lot more than this line.

Reply:
https://github.com/ManageIQ/manageiq-pods/issues/521

Reply:
Actually....

That will need to stay hardcoded because we're trying to avoid RPMs coming from manageiq yum repo. When we add an option to use a different yum repo (#522), that new repo needs to be added here, so it will also be excluded.